pkg/term/tc_other.go

```diff
@@ -1,21 +1,21 @@
 // +build !windows
-// +build !linux !cgo
-// +build !linux !ppc64le
```

Comment:
Or is this also defining tcgets/tcsets for linux as well now? I think it is -- so maybe we should rename the file to tc.go ? *headache*